### ISSUE TITLE
Apple voice navigation updates

### ISSUE THREAD

**Issue by jsandoval81** [csg-org/CompactConnect#913]
From Sprint PR:

#### State & Compact Config UI
1. > It’s not clear that I have to press enter to add an email (no instructions are read aloud): image 4
1. > When an email has already been added, tabbing hovers over the “x” but does not read out the email or otherwise indicate what selecting the “x” would do. (e.g., may require aria-label).

<img width="555" alt="Image" src="https://github.com/user-attachments/assets/f0b63c45-e869-4c37-a032-61f960ade5a5" />

#### Privilege Payment Summary UI
1. > On the privilege purchase cost breakdown page, it reads ("$15.00 x 2" as "15 dollars and zero cents ex two"; the end should probably read "times two").

#### Privilege Select UI
1. > On the privilege purchase [select] page, VoiceOver navigation works oddly on the attestation modals and requires navigating through several "invisible" elements behind the modal before it reads out the contents of the attestation modal itself.

**Comment by jlkravitz** [comment]
Adding to this- it seems like the state/jurisdiction names are not read aloud in Chrome (perhaps also true in other browsers, haven't tested).

<img width="971" height="796" alt="Image" src="https://github.com/user-attachments/assets/6c668cf0-3a7e-49a6-9c97-d3eca22150b3" />

**Comment by jlkravitz** [comment]
One additional item– when navigating over privileges as a state admin, VoiceOver doesn't read out loud which privilege the three-dot menu refers to, making it impossible to know which privilege I'm trying to encumber.

<img width="758" height="356" alt="Image" src="https://github.com/user-attachments/assets/3c005bfd-4f27-4b1d-9deb-e4f5208085f5" />
